.quickstart/quickstart.yml

```diff
@@ -0,0 +1,18 @@
+database_key: salesforce_marketing_cloud_database
+schema_key: salesforce_marketing_cloud_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  salesforce_marketing_cloud__link_enabled:
+    - link_send
+    - link
+  salesforce_marketing_cloud__list_enabled:
+    - list_subscriber
+    - list
+
+destination_configurations:
+  databricks:
+    dispatch:
+      - macro_namespace: dbt_utils
+        search_order: [ 'spark_utils', 'dbt_utils' ]
```

Comment:
New addition of the quickstart.yml where the format was just finalized yesterday.